Cypher.php

```diff
@@ -29,8 +29,14 @@ class Cypher
         'x' => 'B',
         'y' => 'N',
         'z' => 'M'
-    );
-
+    ];
+    
+    /**
+       @method doEncryption
```

Comment:
This isn't the proper use of `method` tags. See https://www.phpdoc.org/docs/latest/references/phpdoc/tags/method.html